tests/test_lightningd.py

```diff
@@ -656,7 +656,7 @@ def test_connect_by_gossip(self):
         l3port = self.node_factory.get_next_port()
         l3 = self.node_factory.get_node(options={"addr": "127.0.0.1:{}".format(l3port)})
```

Comment:
This special casing can be removed now I think?  The code below was added in `tests/utils.py`:

    +            'addr': '127.0.0.1:{}'.format(port),

Reply:
Yes, good point.

Reply:
I just stumbled over this myself, while parallelizing node startups, would be nice to remove this.